### ISSUE TITLE
Allow static sites to be built "automatically"

### ISSUE THREAD

**Issue by zgulde** [zgulde/cods#6]
Like we do for java sites, we just have a `.build_config` that contains the command to build and the output.

We should do this for static sites too.

Example `.build_config` for a static site:

```
BUILD_COMMAND='npm install && npm build'
OUTPUT='dist' # or whatever
```

**Comment by zgulde** [comment]
This was implemented in 30baf9f, but still needs to be documented

**Comment by zgulde** [comment]
documented in d311ae5da558a3dfac3456402d108db77f0f73a1
